### PR TITLE
fixing issue #38

### PR DESCRIPTION
**Pull request by pliguori**
Running-pending-stopped labels are now colored.
Start/Stop buttons are now standard action buttons

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -102,19 +102,28 @@
                             <span ng-if="service.status == 'unknown'" class="service-status-unknown">{{service.name}}</span>
                           </span>
                         </td>
-                        <td class="col-md-1">{{instance.status}}</td>
+                        <td class="col-md-1">
+                          <h4 style="margin:0px;">
+                            <span class="label"
+                                ng-class="['stopped'].indexOf(instance.status.toLowerCase())>=0 ? 'label-danger' : ['running'].indexOf(instance.status.toLowerCase())>=0 ? 'label-success' : 'label-warning'">
```
Comment (FRosner) [inline public/index.html]: This looks like a weird way of testing equality, @pliguori. Please fix! :D

**Comment by FRosner** [comment]
@pliguori can you provide a screenshot of the expected layout, please?

**Comment by pliguori** [comment]
I was trying to comply to your weird tests. I'll change it of course!

**Comment by FRosner** [comment]
Hi @pliguori. Looks good so far, only two comments.
1. I'd like the font inside the status to be normal, not bold
2. I cannot stop a pending instance because the button is disabled

image

**Pull request closed without merging**